### PR TITLE
Mention the serialization changes in API

### PR DESCRIPTION
**Pull request by TheMilek**
| Q               | A
|-----------------|-----
| Branch?         | 2.0 
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| License         | MIT

### CODE REVIEW

File: UPGRADE-API-2.0.md [hunk 1]
```diff
@@ -255,3 +255,228 @@ All the `setter` methods have been removed from the commands above and also ther
 
 1. **ProductOptionValueTranslation**
     - A new resource configuration has been added for the `ProductOptionValueTranslation` resource it is now possible to manage translations for product option values.
+
+## Request Body and Response Updates
+
+### Request Body Changes
+
+#### ChannelPriceHistoryConfig
+Management for `ChannelPriceHistoryConfig` has been moved to the `Channel` resource. The `PUT` request for `Channel` now includes `channelPriceHistoryConfig` fields:
```
Comment (GSadee) [inline UPGRADE-API-2.0.md:264]: Not only PUT, but also POST I believe

File: UPGRADE-API-2.0.md [hunk 2]
```diff
@@ -255,3 +255,228 @@ All the `setter` methods have been removed from the commands above and also ther
...
+
+**Updated `Channel` PUT Request Body:**
+```diff
+    "menuTaxon": "home-accessories",
+  + "channelPriceHistoryConfig": {
+  +  "lowestPriceForDiscountedProductsCheckingPeriod": 30,
```
Comment (GSadee) [inline UPGRADE-API-2.0.md]: Use 4 spaced for indentation

File: UPGRADE-API-2.0.md [hunk 3]
```diff
@@ -255,3 +255,228 @@ All the `setter` methods have been removed from the commands above and also ther
...
+```diff
+     {
+         "@context": "/api/v2/contexts/Zone",
+         "@id": "/api/v2/admin/zones/US",
+         "@type": "Zone",
+    -    "id": 1,
+         "code": "US",
+         "name": "United States",
+    -    "type": "province",
+    +    "type": "country",
```
Comment (GSadee) [inline UPGRADE-API-2.0.md]:
```suggestion
         "type": "country",
```

File: UPGRADE-API-2.0.md [hunk 4]
```diff
@@ -255,3 +255,228 @@ All the `setter` methods have been removed from the commands above and also ther
...
+             "@type": "ShippingMethodTranslation",
+     -       "id": 1,
+             "name": "Standard Shipping",
+             "description": "Delivery within 5-7 business days"
+         }
+     }
+```
+
+#### ProductAssociationType
+`Id` is no longer exposed on the `shop` `get` endpoint.
```
Comment (GSadee) [inline UPGRADE-API-2.0.md]:
```suggestion
`Id` is no longer exposed on the `shop` `GET` endpoint.
```

File: UPGRADE-API-2.0.md (hunk 1 above)
Comment (TheMilek) [reply UPGRADE-API-2.0.md:264]: Only `PUT` in that case, as we were creating it via Channel, but updating via ChannelPriceHistoryConfig

https://github.com/Sylius/Sylius/blob/1.14/src/Sylius/Bundle/ApiBundle/Resources/config/serialization/ChannelPriceHistoryConfig.xml#L25

**Review by GSadee — APPROVED** [review]

**Pull request merged**